libs/watt/src/lib/configuration/watt-date-adapter.ts

```diff
@@ -39,8 +31,16 @@ export class WattDateAdapter extends DateFnsAdapter {
   override getDateNames(): string[] {
     const dateNames = super.getDateNames();
 
-    return this.locale.code === danishLocale
+    return this.locale === danishLocale
```

Comment:
Consideration: Couldn't we just make the replacement regardless of the locale? I know from a micro-optimization perspective it doesn't make sense, but it would remove the hard binding of doing something specific on a specific locale.

Reply:
It did not want to change to much logic around our locales, but only switch date-fns for dayjs. Other refactoring could be done in a other PR :-)
But you do have a good point.

Reply:
Fine by me, I create an issue at the bounty hunters board for a follow-up refactoring task :)